packages/ezresto-directory/ezresto-directory.0.6/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+license: "LGPL-2.1-with-OCaml-exception"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocaml" { >= "4.08" }
+  "dune" { >= "1.7" }
```

Comment:
According to your `dune-project` file this needs to be >= 1.11. Could you modify it everywhere?

Reply:
Thanks for spotting that! I've made the changes and pushed anew.